### ISSUE TITLE
webview object not found on the same domain assigning corewebview2 in  NewWindowRequested event handler

### ISSUE THREAD

**Issue by froggy96** [MicrosoftEdge/WebView2Feedback#1144]
**Description**

**Version**
SDK: 1.0.774.44
Runtime: v4.0.30319
Framework: WinForms
OS: Win10 Pro

**Repro Steps**

1. Make an html to open a new window targeting resource within same domain
ex) window.open("test.html", "_blank");

2. Set corewebview2 object in NewWindowRequested  event.
```
private void CoreWebView2_NewWindowRequested(object sender, CoreWebView2NewWindowRequestedEventArgs e)
{
            e.NewWindow = WebView2.CoreWebView2;
            e.Handled = true;
 }

```
3. Then corewebview2 object contains the new resource (test.html here).

4. Open devtool to check window.chrome.webview object is established --> false!

5. If you change the target on [Step 1] from "_blank" to "_top", window.chrome.webview is established.
6. Is it a bug? or am I doing something wrong?

[error]
![error](https://user-images.githubusercontent.com/11324275/113466008-6c0eef00-9473-11eb-9fdd-7a47e1c17468.PNG)

[ok]
![fine](https://user-images.githubusercontent.com/11324275/113466017-8517a000-9473-11eb-8430-ecf9f3acd324.PNG)

**Additional context**

**Comment by froggy96** [comment]
Another one.
If you open a new window sourcing from OTHER domain (different url such as, http://microsof.com)
It's fine, window.chrome.webview object is established normally.

**Comment by champnic** [comment]
Agreed - this looks like a bug to me as well. I've opened it on our backlog to take a look. Thanks!
